.rspec

```diff
@@ -1,4 +1,3 @@
 --color
---require spec_helper
```

Comment:
Is this removed because it is  defined in `spec/rails_helper.rb` already?

Reply:
Yeah `spec_helper` is already required in `spec/rails_helper.rb`. Also the following error occurs when `--require spec_helper` is present:

```
Failure/Error: class RSpecHtmlMatchers::HaveTag

NameError:
  uninitialized constant RSpecHtmlMatchers
```

Most likely caused by wrong load order